charts/kspm-collector/values.yaml

```diff
@@ -42,7 +42,7 @@ clusterName: ""
 
 image:
   repository: sysdig/kspm-collector
-  tag: 1.24.0
+  tag: 1.25.0
```

Comment:
Did we release version `1.25.0` for kspm-collector or just for the analyzer?

Reply:
we released both versions. the changes here reflect that (see in the list of changed files)